packages/myst-to-jats/src/index.ts

```diff
@@ -520,23 +561,29 @@ class JatsSerializer implements IJatsSerializer {
     this.handlers = opts?.handlers ?? handlers;
     this.mdast = copyNode(mdast);
     if (opts?.extractAbstract) {
-      const abstractMdast = extractPart(this.mdast, 'abstract');
-      if (abstractMdast) {
-        const abstractSerializer = new JatsSerializer(this.file, abstractMdast as Root, {
-          isNotebookArticleRep: this.data.isNotebookArticleRep,
-          slug: this.data.slug,
-          handlers: this.handlers,
-        });
-        abstractSerializer.render();
-        this.data.abstract = abstractSerializer.elements();
-      }
+      const abstractParts = opts.abstractParts ?? [{ part: 'abstract' }];
+      this.data.abstracts = abstractParts
+        .map((def) => renderAbstract(this.file, this.mdast, def, opts))
```

Comment:
This seems fine, we are doing slightly more pre-rendering here, then just grab the outputs later. More consolidated than what we were doing with `getAbstract` before. 👍